src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -113,6 +117,16 @@ public DynamoView(DynamoViewModel dynamoViewModel)
             loginService = new LoginService(this, new System.Windows.Forms.WindowsFormsSynchronizationContext());
             if (dynamoViewModel.Model.AuthenticationManager.HasAuthProvider)
                 dynamoViewModel.Model.AuthenticationManager.AuthProvider.RequestLogin += loginService.ShowLogin;
+
+            var viewExtensions = viewExtensionManager.ExtensionLoader.LoadDirectory(dynamoViewModel.Model.PathManager.ViewExtensionsDirectory);
+            viewExtensionManager.MessageLogged += LogMessage;
+
+            foreach (var ext in viewExtensions)
+            {
+                ext.Startup(null);
+                viewExtensionManager.Add(ext);
```

Comment:
We should surround any invocation of an method on an `Extension` with a `try catch`